src/stores/safariKeyboardStore.ts

```diff
@@ -0,0 +1,37 @@
+import store from './app'
+import reactMinistore from './react-ministore'
+
+const VIRTUAL_KEYBOARD_CLOSE_DURATION = 800
+
+const safariKeyboardStore = reactMinistore<{
+  open: boolean
+  closing: boolean
```

Comment:
`ministore` will infer the generic type from the initialized value, so it's not necessary to explicitly provide it.

Reply:
Fixed by b13e1bc90c7d77215dc6969434d0cb68f9f6bcef